rig/machine_control/tests/test_machine_controller.py

```diff
@@ -330,13 +328,25 @@ def test_load_and_retrieve_routing_tables(self, controller, routes,
             # loaded are present.
             loaded = controller.get_routing_table_entries()
 
-        for route in routes:
-            assert (route, app_id, 0) in loaded
+        for entry in loaded:
+            if entry is not None:
+                (route, app_id, _) = entry
+                assert app_id == 0 or route in routes
 
     def test_app_stop_and_count(self, controller):
```

Comment:
I'd add some [test order constraints](https://github.com/project-rig/rig/blob/master/rig/tests/conftest.py#L85-L103) to make it explicit that this test should occur after doing an app load.